modules/accounts/src/lib.rs

```diff
@@ -4,47 +4,73 @@ use codec::{Decode, Encode};
 use frame_support::{
 	decl_module, decl_storage,
 	dispatch::Dispatchable,
-	traits::{Currency, ExistenceRequirement, Get, OnReapAccount, OnUnbalanced, Time, WithdrawReason},
+	traits::{
+		Currency, ExistenceRequirement, Get, LockIdentifier, LockableCurrency, OnReapAccount, OnUnbalanced, Time,
+		WithdrawReason, WithdrawReasons,
+	},
 	weights::DispatchInfo,
 	IsSubType, Parameter,
 };
 use orml_traits::MultiCurrency;
 use rstd::prelude::*;
 use sp_runtime::{
-	traits::{SaturatedConversion, Saturating, SignedExtension, Zero},
+	traits::{Bounded, SaturatedConversion, Saturating, SignedExtension, Zero},
 	transaction_validity::{
 		InvalidTransaction, TransactionPriority, TransactionValidity, TransactionValidityError, ValidTransaction,
 	},
 };
+use system::ensure_signed;
 
 mod mock;
 mod tests;
 
+const ACCOUNTS_ID: LockIdentifier = *b"accounts";
```

Comment:
better to prefix those with `ACA`

```suggestion
const ACCOUNTS_ID: LockIdentifier = *b"ACA/acct";
```